main.go

```diff
@@ -192,6 +215,22 @@ func getSessionFromLogin(oktaCfg *OktaConfig) (string, error) {
 		return "", errors.New("Must supply a username and password")
 	}
 
+	sessionToken, err := tryLogin(oktaCfg, user, pass)
+	if err == nil && sessionToken != "" {
+		keystore.AddPassword(APPNAME, CREDENTIALS_USERNAME, user)
+		if err != nil {
+			debug("err storing username", err)
```

Comment:
this debug function, which I need to replace, operates like printf, so these will need to read something like `debug("err storing username: %s", err)`